lib/bridge/IrcBridge.js

```diff
@@ -152,6 +152,45 @@ IrcBridge.prototype._initialiseMetrics = function() {
         help: "Histogram of processing durations of received remote messages",
         labels: ["outcome"],
     });
+
+    // Custom IRC metrics
+    const reconnQueue = metrics.addGauge({
+        name: "clientpool_reconnect_queue",
```

Comment:
I'm worried that we might be trying to over generalize this though. The concept of an IRC reconnection pool where TCP connections are trying to regain consciousness and a twitter/gitter/discord connection pool where we don't really handle connections in the same way.

But I guess from a raw reporting metric standpoint, we don't really care because the context should be obvious?

Reply:
This feels fairly IRC specific. Some bridges have a reconnect concept, although there's still too many that don't for me to feel comfortable.